### PR TITLE
LPD-29389 Apply Users and Organization changes to production only

### PR DESCRIPTION
**Pull request by brookedalton**
https://liferay.atlassian.net/browse/LPD-29389

### CODE REVIEW

**Comment by pei-jung** [comment]
@noornajj can you review this PR? Thanks!

**Comment by noornajj** [comment]
ci:test:relevant

**Comment by noornajj** [comment]
ci:test:publications

File: modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/change/tracking/test/UsersAdminPortletTest.java [hunk 1]
```diff
@@ -0,0 +1,1009 @@
...
+
+	@Test
+	public void testNoCTEntryForAddedEditUserMVCAction() throws Exception {
+		String addUserScreenName = RandomTestUtil.randomString();
+
+		try (SafeCloseable safeCloseable =
+				CTCollectionThreadLocal.setCTCollectionIdWithSafeCloseable(
+					_ctCollection.getCtCollectionId())) {
+
+			_portlet.processAction(
```
Comment (noornajj) [inline modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/change/tracking/test/UsersAdminPortletTest.java:112]: Hey Brooke! Great job!
I would just include this in the `_getActionRequest` method and maybe rename it to `_processActionRequest`.

File: modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/change/tracking/test/UsersAdminPortletTest.java [hunk 2]
```diff
@@ -0,0 +1,1009 @@
...
+						"emailAddress",
+						RandomTestUtil.randomString() + "@liferay.com"
+					).put(
+						"firstName", RandomTestUtil.randomString()
+					).put(
+						"lastName", RandomTestUtil.randomString()
+					).put(
+						"screenName", addUserScreenName
+					).build()),
+				new CustomMockLiferayPortletActionResponse());
```
Comment (noornajj) [inline modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/change/tracking/test/UsersAdminPortletTest.java:133]: Same with this line

**Review by noornajj — CHANGES_REQUESTED** [review]

**Review by noornajj — CHANGES_REQUESTED** [review]
Hey Brooke! Great job!
Just one thing you can extract into the _getActionRequest method

**Comment by noornajj** [comment]
Hey @brookedalton 
The failures in the publications suite may be related to your changes. Depending on the tests you may need to rewwrite the tests or maybe even remove them.
Can you take a look?

File: modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/change/tracking/test/UsersAdminPortletTest.java (hunk 2 above)
Comment (brookedalton) [reply modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/change/tracking/test/UsersAdminPortletTest.java:133]: Hey Noor! Would it make sense to also include the 
```
 try (SafeCloseable safeCloseable =
				CTCollectionThreadLocal.setCTCollectionIdWithSafeCloseable(
					_ctCollection.getCtCollectionId())
```
  in the "_processActionRequest" ?

Comment (noornajj) [reply modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/change/tracking/test/UsersAdminPortletTest.java:133]: Yeah, definitely! 
I would pass in the ctCollectionId into the method as a parameter though. That increases the method's usability for other ctCollections

**Comment by brookedalton** [comment]
ci:test:sf

**Comment by brookedalton** [comment]
ci:test:publications

**Comment by brookedalton** [comment]
The failures above are not caused by my changes and are now failing upstream

**Comment by brookedalton** [comment]
ci:test:relevant

**Comment by brookedalton** [comment]
ci:forward:force

File: modules/apps/image-uploader/image-uploader-web/src/main/java/com/liferay/image/uploader/web/internal/portlet/action/UploadImageMVCActionCommand.java [hunk 3]
```diff
@@ -100,7 +102,13 @@ protected void doProcessAction(
 
 		long maxFileSize = UploadImageUtil.getMaxFileSize(actionRequest);
 
-		try {
+		long ctCollectionId = ParamUtil.getLong(
+			actionRequest, "ctCollectionId");
```
Comment (pei-jung) [inline modules/apps/image-uploader/image-uploader-web/src/main/java/com/liferay/image/uploader/web/internal/portlet/action/UploadImageMVCActionCommand.java:106]: Where is this set? I don't see the usage.

**Comment by brookedalton** [comment]
resent: https://github.com/liferay-site-management/liferay-portal/pull/1363

**Pull request closed without merging**